scheduler/cloudformation/template.go

```diff
@@ -17,9 +18,17 @@ import (
 	"github.com/remind101/empire"
 	"github.com/remind101/empire/pkg/arn"
 	"github.com/remind101/empire/pkg/bytesize"
+	"github.com/remind101/empire/pkg/troposphere"
 	"github.com/remind101/empire/scheduler"
 )
 
+var (
```

Comment:
As part of this, I ended up cleaning up some of the template building by replicating some of [troposphere](https://github.com/cloudtools/troposphere) in Go. If it gets cumbersome to review, I can pull that out into a separate PR and rebase this.